### ISSUE TITLE
Error when calling disconnectAll

### ISSUE THREAD

**Issue by pietermees** [grantila/fetch-h2#12]
When calling `disconnectAll`
``` 
  TypeError {
    message: 'Cannot read property \'destroy\' of undefined',
  }

  Context.handleDisconnect (node_modules/fetch-h2/lib/context.ts:279:11)
  Context.disconnectAll (node_modules/fetch-h2/lib/context.ts:323:24)
```

I think the code assumes that a session was actually created, while it might have failed with an error or could still be in the process of being created?

**Comment by grantila** [comment]
I think this is fixed in 0.2.3, please try again.

**Comment by pietermees** [comment]
Works, thanks for the quick fix!
